packages/core/src/Stores/notification-store.js

```diff
@@ -331,7 +331,10 @@ export default class NotificationStore extends BaseStore {
 
         let has_missing_required_field;
 
-        if (website_status?.message?.length) {
+        const { site_status = '' } = website_status;
+        const isServerDown = site_status === 'down' || site_status === 'updating';
```

Comment:
[Suggestion] - It would be nice to make this into a generic function and reuse it since it is also being defined in line 324.

also since we are still coding in deriv-app standards then a rename is needed for `is_server_down` :)

Reply:
down for both~ @matin-deriv